drivers/display/Kconfig.sdl

```diff
@@ -62,6 +62,14 @@ config SDL_DISPLAY_MONO_MSB_FIRST
 	  If selected, set the MSB to represent the first pixel.
 	  This applies when the pixel format is MONO01/MONO10.
 
+config SDL_DISPLAY_MONO_VTILED
```

Comment:
Please add the htiled test condition to `testcase.yaml`.

Reply:
As far as I can see there doesn't seem to be any other issues.

Reply:
Done